app/views/movies/_movie_production_companies.html.erb

```diff
@@ -1,8 +1,8 @@
-<% if @movie.production_companies.present? %>
```

Comment:
I'm switching from instance variables to locals in all of these movie content supporting partials.